### ISSUE TITLE
Crash in listen when running guard on Catalina MacOS

### ISSUE THREAD

**Issue by pitosalas** [guard/listen#467]
At least I think it's caused by listen:

```
/Users/pitosalas/.rbenv/versions/2.6.2/lib/ruby/gems/2.6.0/gems/guard-2.15.1/lib/guard/commander.rb:56:in `stop': undefined method `stop' for nil:NilClass (NoMethodError)
	from /Users/pitosalas/.rbenv/versions/2.6.2/lib/ruby/gems/2.6.0/gems/guard-2.15.1/lib/guard/commander.rb:52:in `ensure in start'
	from /Users/pitosalas/.rbenv/versions/2.6.2/lib/ruby/gems/2.6.0/gems/guard-2.15.1/lib/guard/commander.rb:52:in `start'
	from /Users/pitosalas/.rbenv/versions/2.6.2/lib/ruby/gems/2.6.0/gems/guard-2.15.1/lib/guard/cli/environments/valid.rb:16:in `start_guard'
	from /Users/pitosalas/.rbenv/versions/2.6.2/lib/ruby/gems/2.6.0/gems/guard-2.15.1/lib/guard/cli.rb:122:in `start'
	from /Users/pitosalas/.rbenv/versions/2.6.2/lib/ruby/gems/2.6.0/gems/thor-0.20.3/lib/thor/command.rb:27:in `run'
	from /Users/pitosalas/.rbenv/versions/2.6.2/lib/ruby/gems/2.6.0/gems/thor-0.20.3/lib/thor/invocation.rb:126:in `invoke_command'
	from /Users/pitosalas/.rbenv/versions/2.6.2/lib/ruby/gems/2.6.0/gems/thor-0.20.3/lib/thor.rb:387:in `dispatch'
	from /Users/pitosalas/.rbenv/versions/2.6.2/lib/ruby/gems/2.6.0/gems/thor-0.20.3/lib/thor/base.rb:466:in `start'
	from /Users/pitosalas/.rbenv/versions/2.6.2/lib/ruby/gems/2.6.0/gems/guard-2.15.1/lib/guard/aruba_adapter.rb:32:in `execute'
	from /Users/pitosalas/.rbenv/versions/2.6.2/lib/ruby/gems/2.6.0/gems/guard-2.15.1/lib/guard/aruba_adapter.rb:19:in `execute!'
	from /Users/pitosalas/.rbenv/versions/2.6.2/lib/ruby/gems/2.6.0/gems/guard-2.15.1/bin/_guard-core:11:in `<main>'
```

**Comment by ColinDKelley** [comment]
@pitosalas Much of the `listen` code has been restructured in v3.3, so this may be fixed. Although I can't tell which part of the above involved the `listen` gem, if any.

Any chance you can try listen `v3.3.0.pre.2` and let us know if the problem persists?

**Comment by ColinDKelley** [comment]
I'm going to assume this has been fixed at least by v3.3.0. If the issue reappears, we can open a fresh ticket.
